lib/containers/root/Root.js

```diff
@@ -0,0 +1,24 @@
+/*
+ * == BSD2 LICENSE ==
+ * Copyright (c) 2015, Tidepool Project
+ *
+ * This program is free software; you can redistribute it and/or modify it under
+ * the terms of the associated License, which is identical to the BSD 2-Clause
+ * License as published by the Open Source Initiative at opensource.org.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the License for more details.
+ *
+ * You should have received a copy of the License along with this program; if
+ * not, you can obtain one from Tidepool Project at tidepool.org.
+ * == BSD2 LICENSE ==
+ */
+
+/* global __DEBUG__ */
+
+if (__DEBUG__ === true) {
```

Comment:
We're actually going to break this out further, because the redux dev tools are a *huge* performance suck and the logger especially is very very noisy if you working on devices (updates to the progress bar fire an action for *every* update). So we're probably going to have a collection of configs: `__DEBUG__`, `__REDUX_LOG__`, `__REDUX_DEV_UI__` etc.